fabfile/tpch_confs/tpch_default.ini

```diff
@@ -1,6 +1,6 @@
 [test]
-use_enterprise: on
-postgres_citus_versions: [('12.1', 'enterprise-master'), ('12.1', 'release-9.2')]
+use_enterprise: off
+postgres_citus_versions: [('14.3', 'release-11.0')]
```

Comment:
maybe we should use pg 14.5

Reply:
(and two other files too)

Reply:
Also I will turn the enterprise flags off for now.
Depending on how much work might there be, I can also clean up the enterprise logic, maybe in another PR.